src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/parameters/Base64EncodedStringParameterValue/Base64EncodedStringParameterValue.jelly

```diff
@@ -27,6 +27,9 @@ THE SOFTWARE.
         xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form"
         xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project">
         <f:entry title="${it.name}" description="${it.description}">
+            <div name="parameter" description="${it.description}">
```

Comment:
Hi, sorry for the late reply. I have been busy with some other things.

I have also done some searching. There are two other projects that I found providing Rebuild plugin support the exact same way as we do.

[matrix-combinations-plugin](https://github.com/jenkinsci/matrix-combinations-plugin)
* [MatrixCombinationsRebuildParameterProvider.java](https://github.com/jenkinsci/matrix-combinations-plugin/blob/master/src/main/java/hudson/plugins/matrix_configuration_parameter/MatrixCombinationsRebuildParameterProvider.java)
* [rebuild.groovy](https://github.com/jenkinsci/matrix-combinations-plugin/blob/master/src/main/resources/hudson/plugins/matrix_configuration_parameter/MatrixCombinationsParameterValue/rebuild.groovy)

[promoted-builds-plugin](https://github.com/jenkinsci/promoted-builds-plugin)
* [PromotedBuildRebuildParameterProvider.java](https://github.com/jenkinsci/promoted-builds-plugin/blob/master/src/main/java/hudson/plugins/promoted_builds/parameters/PromotedBuildRebuildParameterProvider.java)
* [value.jelly](https://github.com/jenkinsci/promoted-builds-plugin/blob/master/src/main/resources/hudson/plugins/promoted_builds/parameters/PromotedBuildParameterValue/value.jelly)

I we revert this PR, adding `Base64EncodedStringParameterValue.jelly` to the resource of `RebuildAction` in Rebuild plugin is not enough. A parameter provider class is needed in rebuild too. However, Rebuild doesn't really need such a provider class. So I don't think there is any problem adding changes to Gerrit Trigger.